spark/dl/src/test/scala/com/intel/analytics/bigdl/keras/KerasRunner.scala

```diff
@@ -109,11 +109,18 @@ object KerasRunner {
                               keyName: String): Array[Tensor[Float]] = {
     if (!pvalues.keySet.filter(key => key.contains(keyName)).isEmpty) {
       val weightNum = pvalues.keySet.filter(key => key.contains(keyName)).size / 2
-      Range(0, weightNum).map {i =>
-        Tensor[Float](
-          data = pvalues(s"${keyName}_${i}_value"),
-          shape = pvalues(s"${keyName}_${i}_shape").map(_.toInt))
-      }.toArray
+      if (weightNum > 1) { // with bias
```

Comment:
why changing this?

Reply:
I discussed this in the previous PR into your branch.
If the layer has no bias,
data = pvalues(s"${keyName}_${i}_value") will get an error as there's only 1 weight, it will be just ${keyName} instead of ${keyName}_0
Changing this will take care of the case if there's no bias.

Reply:
maybe we can change the logic that generating the `pvalues` to unify the naming.

Reply:
Yeah. This is only one option of change. We can discuss tomorrow.

Reply:
val keyPrefix = if (weightNum > 1) {${keyName}_${i}} else {${keyName}}
key = keyPrefix + "_value"